Git shortcuts

```diff
@@ -0,0 +1,8 @@
+commit
```

Comment:
Pra que serve esse arquivo?
Por favor, nao crie nomes de arquivo com espacos, isso causa cancer
Se isso for apenas um txt, seria legal usar a extencao txt. Ou melhor ainda, poderia usar a extencao `.md` e usar a syntaxe de markdown do github, como o README.md

De uma olhada em https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet
Eh bem simples usar, nao requer pratica nem sequer experiencia